docs/ops/deployment/docker.md

```diff
@@ -334,7 +334,7 @@ The next chapters show examples of configuration files to run Flink.
 * Access the *Flink Master* container
 
     ```sh
-    docker exec -it $(docker ps --filter name=jobmanager --format={{.ID}}) /bin/sh
+    docker exec -it $(docker ps --filter name=jobmanager --format={% raw  %}{{.ID}}{% endraw  %}) /bin/sh
```

Comment:
```suggestion
    docker exec -it $(docker ps --filter name=jobmanager --format={% raw %}{{.ID}}{% endraw %}) /bin/sh
```
There are a bunch of double-spaces.

Reply:
Sorry for that, missed it when copying from jekyll's docs. Will remove it while merging.